### ISSUE TITLE
*.mosl.txt files - comma in comment lines cause read error in Reaper

### ISSUE THREAD

**Issue by lossius** [ambisonictoolkit/atk-matrices#12]
I have discovered a weird shortcoming when reading `*.mosl.txt` files in Reaper. If a comment line contains a comma (`,`), the file will not be read by Reaper. This cause problem with all matrices in v.1.0 as a comma is used consistently in the `// Dimensions: rows, columns` line in the header section:

```
// set : FOA

// type : decoder

// op : matrix

// kind : hoa1

// note : A first order Ambisonic format exchange decoder.

// author : Ambisonic Toolkit

// Dimensions: rows, columns
4
4
```

@joslloand : The easiest solution is to just rid ourselves of that comma, changing the line to 

```
// Dimensions: rows columns
```

Two questions:

1. Are these files _only_ used in Reaper? If so, it should be straightforward to remove the comma
2. Are they autogenerated from the YAML file? If so the code for generating them also need to be updated

This issue relates to https://github.com/ambisonictoolkit/atk-reaper/issues/92

**Comment by joslloand** [comment]
> 1. Are these files only used in Reaper? If so, it should be straightforward to remove the comma

@lossius, ATK-SC3 can read AND write the .mosl files.

The writing is handled by [ATKMatrix:-prWriteMatrixToMOSL](https://github.com/ambisonictoolkit/atk-sc3/blob/master/Classes/ATKMatrix.sc#L178-L198). And, actually, the line in question is [here](https://github.com/ambisonictoolkit/atk-sc3/blob/master/Classes/ATKMatrix.sc#L184).

This _could_ be updated to remove the comma, OR `//` could be replace by `#` or `;`, the JSFX comment character as stated by Justin Frankel. If removing the comma would work equally as well, this could be a quick fix.

Reading is manage by [FoaMatrix:-prParseMOSL](https://github.com/ambisonictoolkit/atk-sc3/blob/master/Classes/FoaMatrix.sc#L409-L459). If we want to migrate to the use of `#` or `;` as comment characters, these would have to be added [here](https://github.com/ambisonictoolkit/atk-sc3/blob/master/Classes/FoaMatrix.sc#L421).

My feeling is that recasting reading/writing comments to the JSFX convention would be a sensible option... and we should file a bug against atk-sc3.

@mtmccrea, any thoughts?

**Comment by joslloand** [comment]
> 2. Are they autogenerated from the YAML file? If so the code for generating them also need to be updated

Good question... will need to review....

**Comment by lossius** [comment]
My preference is to change comment syntax from `//` to `#`, provided that it does not cause problems for any of yo in SuperCollider.

**Comment by joslloand** [comment]
@lossius, ok, good to know. At the moment I think it shouldn't be too tricky to update... but will need some care.

I'll prioritize the problem...

**Comment by joslloand** [comment]
fixed: 13e69ea714d883731411cd5508d65418c5bffd15
